src/Chain.hs

```diff
@@ -157,14 +158,18 @@ headBlockNo (_ :> b) = blockNo b
 
 -- | Produce the list of blocks, from most recent back to genesis
 --
-toList :: Chain block -> [block]
-toList = foldChain (flip (:)) []
+toNewestFirst :: Chain block -> [block]
+toNewestFirst = foldChain (flip (:)) []
```

Comment:
Do we want newtype wrappers for the oldest first and newest first? I think it was not that bad idea to keep this from the old code base, though I didn't had to deal with them that much.

Reply:
Yeah, that might eventually not be a a bad idea, though it might be something we might want to do uniformly across all repos?